BaseClasses.py

```diff
@@ -1206,7 +1206,10 @@ def hint_text(self) -> str:
 class ItemClassification(IntFlag):
     filler = 0b0000  # aka trash, as in filler items like ammo, currency etc,
     progression = 0b0001  # Item that is logically relevant
-    useful = 0b0010  # Item that is generally quite useful, but not required for anything logical
+    # Protects this item from being placed on excluded or unreachable locations.
+    useful = 0b0010  # Item that is especially useful.
+    # Protects this item from being placed on excluded or unreachable locations.
+    # When combined with another flag like "progression", it means "an especially useful progression item".
     trap = 0b0100  # detrimental item
```

Comment:
To  communicate better about combining `trap` with other items, maybe "item that is detrimental in some way"?
(because it might not be completely detrimental overall)

Reply:
I like this suggestion